src/spec-configuration/containerFeaturesOrder.ts

```diff
@@ -49,12 +49,36 @@ export function computeInstallationOrder(features: FeatureSet[]) {
         feature,
         before: new Set(),
         after: new Set(),
-    })).reduce((map, feature) => map.set(feature.feature.sourceInformation.userFeatureId, feature), new Map<string, FeatureNode>());
+    })).reduce((map, feature) => map.set(feature.feature.sourceInformation.userFeatureId.split(':')[0], feature), new Map<string, FeatureNode>());
```

Comment:
The code below for `for (const firstId of later.feature.features[0].installsAfter || []) {` was broken & wasn't having any effect on the computed order.

the installsAfter has id without version, however, this line was setting id with version. Hence, 

> let first = nodesById.get(firstId);

never returned any item. Fixed that.

Reply:
Also reported in https://github.com/devcontainers/cli/issues/345